apikeys/index.js

```diff
@@ -35,6 +37,10 @@ module.exports.init = function(config, logger, stats) {
 		var keepApiKey = config.hasOwnProperty('keep-api-key') ? config['keep-api-key'] : false;
 		//cache api keys
         cacheKey = config.hasOwnProperty("cacheKey") ? config.cacheKey : false;
+        //cache ttl
+        cacheKeyTTL = config.hasOwnProperty("cacheKeyTTL") ? config.cacheKeyTTL : 60000;
```

Comment:
doesn't respect the default cacheKeyTTL value prev set on line 25 - avoid magic numbers. same for cacheSize